docs/en/connector-v2/source/MongoDB.md

```diff
@@ -0,0 +1,82 @@
+# MongoDb
+
+> MongoDb source connector
+
+## Description
+
+Read data from MongoDB.
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [x] [stream](../../concept/connector-v2-features.md)
```

Comment:
Are you sure mongo source support stream mode? I think if we support stream mode need implement mongo CDC.